x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/ssl/SSLErrorMessageFileTests.java

```diff
@@ -235,14 +236,21 @@ private void checkMissingResource(String sslManagerType, String fileType, String
         final String key = prefix + "." + configKey;
         settings.put(key, fileName);
 
+        final String fileErrorMessage = "cannot read configured " + fileType + " [" + fileName + "] because the file does not exist";
         Throwable exception = expectFailure(settings);
         assertThat(exception, throwableWithMessage("failed to load SSL configuration [" + prefix + "]"));
         assertThat(exception, instanceOf(ElasticsearchSecurityException.class));
 
         exception = exception.getCause();
-        assertThat(exception, throwableWithMessage(
-            "failed to initialize SSL " + sslManagerType + " - " + fileType + " file [" + fileName + "] does not exist"));
-        assertThat(exception, instanceOf(ElasticsearchException.class));
+        // This is needed temporarily while we're converting from X-Pack SSL to libs/ssl-config
```

Comment:
Because we've switched the PEM file reading over to ssl-config, but other scenarios are still using X-Pack, there's a mix of error messages in use.
This `if` is a temporary measure to keep this PR small, until we unify everything to depend on ssl-config.